config/routes.rb

```diff
@@ -177,9 +177,9 @@
     get :sample_locations, on: :collection
   end
 
-  authenticate :auth0_user, ->(u) { u.admin? } do
-    mount RESQUE_SERVER, at: "/resque"
-  end
+  # authenticate :auth0_user, ->(u) { u.admin? } do
```

Comment:
Nit: Could you add a comment here as well? To explain why this is commented out.

Reply:
👍